k8s/airflow-operator/Makefile

```diff
@@ -0,0 +1,66 @@
+include ../vendor/marketplace-tools/app.Makefile
+include ../vendor/marketplace-tools/crd.Makefile
+include ../vendor/marketplace-tools/gcloud.Makefile
+include ../vendor/marketplace-tools/marketplace.Makefile
+include ../vendor/marketplace-tools/ubbagent.Makefile
```

Comment:
Please do not treat it as a blocker for this PR, but ubbagent is not used in this app - the same as in other click-to-deploy apps. Maybe we should remove it? @huyhg please share your thoughts on this.

Reply:
done